### ISSUE TITLE
Support for dataframe.duplicated method

### ISSUE THREAD

**Issue by dirkschneemann** [dask/dask#1854]
Would it be possible to also include pandas [dataframe.duplicated() method](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.duplicated.html) in the dask.dataframe API? 

[drop_duplicates() is already included](http://dask.pydata.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.drop_duplicates) and I assume that they work pretty similar internally (please correct me if I'm wrong), so this should hopefully not be one of the operations that are not supported due to technical or speed limitations.

**Comment by mrocklin** [comment]
duplicated and drop_duplicated would not work similarly in parallel.  Implementing `duplicated` in parallel sounds difficult to me.

**Comment by mrocklin** [comment]
Closing as out of scope

**Comment by heathermrauch** [comment]
If this is out of scope, how do you propose checking a large dataframe for duplicates using dask?

**Comment by mrocklin** [comment]
I haven't made such a proposal for how to determine if duplicates exist.  I apologize if I implied otherwise.

The next thing to do here would be to come up with a feasible parallel algorithm.  If you'd like to help do some research there that would probably be the best way to push this forward.
